ChangeLog

```diff
@@ -1,3 +1,9 @@
+13.0.0:
+ - Removed support for v5.
+ - Updated get_ad_group_bid_modifier example to limit number of results in query.
```

Comment:
We don't need to mention any of this in the changelog. The examples aren't bundled into the gem.

Reply:
To be clear I meant the last 3 lines of the entry. I had them highlighted but I forgot that GitHub doesn't work that way :)

Reply:
Yup makes sense :) I've removed the 3 lines about examples.